File: src/main/java/com/ysmull/easeshop/model/vo/UserVO.java
package com.ysmull.easeshop.model.vo;

/**
 * @author maoyusu
 */
public class UserVO {

    private Long userId;
    private String username;
    private Integer role;

    public Long getUserId() {
        return userId;
    }

    public void setUserId(Long userId) {
        this.userId = userId;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public Integer getRole() {
        return role;
    }

    public void setRole(Integer role) {
        this.role = role;
    }

    @Override
    public String toString() {
        return "UserVO{" +
                "userId=" + userId +
                ", username='" + username + '\'' +
                ", role=" + role +
                '}';
    }
}

File: EaseMall-biz/src/main/java/com/ysmull/easemall/model/vo/UserVO.kt
package com.ysmull.easemall.model.vo

/**
 * @author maoyusu
 */
class UserVO {

    var userId: Long? = null
    var username: String? = null
    var role: Int? = null

    override fun toString(): String {
        return "UserVO{" +
                "userId=" + userId +
                ", username='" + username + '\''.toString() +
                ", role=" + role +
                '}'.toString()
    }
}
